### PR TITLE
Add gradle 2.2.1

### PR DESCRIPTION
**Pull request by l4u**

### CODE REVIEW

File: gradle221.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require "formula"
```
Comment (DomT4) [inline gradle221.rb]: :scissors: this line :)

File: gradle221.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+class Gradle221 < Formula
+  homepage "http://www.gradle.org/"
+  url "https://services.gradle.org/distributions/gradle-2.2.1-bin.zip"
+  sha1 "97ae081bc8be8b691e21dcf792eff1630dbc8eb6"
```
Comment (DomT4) [inline gradle221.rb]: ` sha256 ` please.

**Comment by DomT4** [comment]
Couple of nits, but otherwise looks great :+1:

**Comment by l4u** [comment]
Updated with sha256

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, we appreciate it :smiley_cat:.

**Pull request closed without merging**